src/tasks/AppleAppBuilder/Templates/runtime.m

```diff
@@ -261,8 +236,29 @@ static int32_t load_icu_data ()
     const char* bundle = get_bundle_path ();
     chdir (bundle);
 
+    char icu_dat_path [1024];
+    int res;
+
+    res = snprintf (icu_dat_path, sizeof (icu_dat_path) - 1, "%s/%s", bundle, "icudt.dat");
+    assert (res > 0);
+
     // TODO: set TRUSTED_PLATFORM_ASSEMBLIES, APP_PATHS and NATIVE_DLL_SEARCH_DIRECTORIES
-    monovm_initialize(0, NULL, NULL);
+    const char *appctx_keys [] = {
+        "RUNTIME_IDENTIFIER", 
```

Comment:
Curious, I don't see any place on the repo using this app context switch, what is this for?

Reply:
Good question.  I don't know.  I brought it over because it was being used in our android setup.

Reply:
It is used here: https://github.com/dotnet/runtime/blob/main/src/libraries/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/RuntimeInformation.cs#L50

Reply:
Ah, my search was busted (thanks GH search). We use it for `RuntimeInformation.RuntimeIdentifier`: https://github.com/dotnet/runtime/blob/79ae74f5ca5c8a6fe3a48935e85bd7374959c570/src/libraries/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/RuntimeInformation.cs#L49-L50